module3/lessons/Week5/IntroToBootstrap.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: page
+title: Intro to Bootstrap
+---
+
+## Learning Goals
+- 
+
+We will build on our foundational understanding of [HTML](/module3/lessons/week1/HTMLFundamentals) and [CSS](/module3/lessons/week1/CSSFundamentals).
+
+## Warm Up
+
+<section class='call-to-action' markdown='1'>
+
+If necessary, re-familiarize yourself with earlier lessons:
+* [HTML Fundamentals](/module3/lessons/week1/HTMLFundamentals)
+* [CSS Fundamentals](/module3/lessons/week1/CSSFundamentals)
+
+## What is Bootstrap?
+
+Bootstrap is a popular CSS Framework used to develop responsive and mobile-first websites, based on a grid design. Most everything is built in a container, with rows and columns nested inside.
+
+Most importantly, it is **FREE** to use.
+
+## How to use Bootstrap
+
+You can use Bootstrap in one of two ways:
```

Comment:
Is there a preferred method for using bootstrap?  I have always used a CDN - are we going to ask students to try both ways?

Reply:
I think the CDN is the preferred method and this is how students should use it. Installing Bootstrap is for informational purposes only.